app/views/patients/_call_log.html.erb

```diff
@@ -2,10 +2,10 @@
   <div class="margin-bottom col-sm-12">
     <h2>
       Call Log
-      <small><%= link_to "Record new call <span class='glyphicon glyphicon-earphone' aria-hidden='true'></span><span class='sr-only'>Call</span>".html_safe, "#call-#{patient.primary_phone_display}", data: {toggle: "modal"} %></small>
+      <small><%= link_to "Record new call <span class='glyphicon glyphicon-earphone' aria-hidden='true'></span><span class='sr-only'>Call</span>".html_safe, "#call-#{@patient.primary_phone_display}", data: {toggle: "modal"} %></small>
```

Comment:
Looks like the changes to this file are turning our local into an instance variable, and that's because if it's a local then the call chokes as follows.

```
MONGODB | localhost:27017 | dcaf_case_management_development.update | SUCCEEDED | 0.000512s
  Rendered patients/_call_log.html.erb (73.1ms)
  Rendered calls/create.js.erb (75.1ms)
Completed 500 Internal Server Error in 97ms

NameError - undefined local variable or method `patient' for #<#<Class:0x007f9b2aea7370>:0x007f9b2e1eaf98>
Did you mean?  @patient:
  app/views/patients/_call_log.html.erb:5:in `_app_views_patients__call_log_html_erb___1989415623033319292_70152230704360'

```

I'd prefer to have this as a local variable rather than an instance variable, but can live without it if it's a nontrivial change. What are your thoughts @tingaloo ?